client/ayon_core/hosts/photoshop/plugins/create/create_image.py

```diff
@@ -53,7 +53,7 @@ def create(self, product_name_from_ui, data, pre_create_data):
             stub.select_layers(stub.get_layers())
             try:
                 group = stub.group_selected_layers(product_name_from_ui)
-            except:
+            except:  # noqa E722
```

Comment:
This may need to be a `ValueError` check if it's [this error](https://github.com/ynput/ayon-core/blob/78a895b720fda253a37725ce30814ca85abafdb9/client/ayon_core/hosts/photoshop/api/ws_stub.py#L292) but I'm not sure. Maybe @kalisp knows or can confirm?

Reply:
Yep, modified.